test/derivative/CudaGeneralDeriv.cu

```diff
@@ -1,13 +1,20 @@
 #include <string>
 #include <chrono>
+#include <iostream>
+#include <fstream>
 #include "CudaGeneralDeriv.h"
 #include "ClassicDeriv.h"
+#ifdef USE_COMPILED
+#include "general.cu"
+#include "general_flipped.cu"
```

Comment:
I wonder whether we should include the CUDA files here or link the object files later?